### PR TITLE
Mailchimp docs

### PR DESCRIPTION
**Pull request by erin2722**

### CODE REVIEW

**Review by ivov — COMMENTED** [review]
A commit on Discord docs got mixed in here!

**Review by ralph-dev — APPROVED** [review]
LGTM

**Comment by erin2722** [comment]
Comment-- I accidentally merged in the discord docs to this branch but undid it with the most recent commit, so we are all good 👍

**Review by ivov — APPROVED** [review]

File: docs/nodes/nodes-library/nodes/Mailchimp/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Basic Operations
+
+- Create, delete, update, and get users on your marketing lists.
+- Add and delete user tags on your marketing list.
+
+## Example Usage
+
+This workflow demonstrates creating a new member for a list. You can also find Mailchimp [workflows](https://n8n.io/integrations/n8n-nodes-base.mailchimp) on this website. This example usage workflow would use the following two nodes.
+- [Start]()
+- [Mailchimp]()
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Mailchimp/README.md]: We need to set the links as we did with Discord.

**Review by ivov — CHANGES_REQUESTED** [review]

File: docs/nodes/nodes-library/nodes/Mailchimp/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Basic Operations
+
+- Create, delete, update, and get users on your marketing lists.
+- Add and delete user tags on your marketing list.
+
+## Example Usage
+
+This workflow demonstrates creating a new member for a list. You can also [find Mailchimp workflows on this website](https://n8n.io/integrations/n8n-nodes-base.mailchimp). This example usage workflow uses the following two nodes.
+
+- [Start](../../core-nodes/Start)
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Mailchimp/README.md:23]: That link points to a dir. I think it should be `(../../core-nodes/Start/README.md)`

Comment (erin2722) [reply docs/nodes/nodes-library/nodes/Mailchimp/README.md:23]: Both links work because like index, the dir automatically points to the readme.

Comment (ivov) [reply docs/nodes/nodes-library/nodes/Mailchimp/README.md:23]: I see, perfect then!

**Pull request merged**